### PR TITLE
Create the Chromium Chronicle Episode 7

### PR DESCRIPTION
**Pull request by enegiar**
What's changed, or what was fixed?
-Added the content and formatting for The Chromium Chronicle Episode 7, based on this doc:
https://docs.google.com/document/d/1BAbilhrYl7Cgg3FnjtgisnEBGVx7h3bP3-1C3l7SnBQ/edit#

**Target Live Date:** 2019-10-22

- [ ] This has been reviewed and approved by (NAME)
- [ ] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by enegiar** [comment]
> **Whoops!**
> 
> There were _1 warnings_ that will prevent this PR from being merged. Please take a look, and either fix, or provide a justification for why they can't be fixed.
> 
> **WARNINGS**
> [`src/content/en/updates/2019/10/chromium-chronicle-7.md#L5`](https://github.com/Google/WebFundamentals/blame/f55574a91f0c24a5c6e5cb0981c0c0baab4ff742/src/content/en/updates/2019/10/chromium-chronicle-7.md#L5) - WF Tag 'wf_updated_on' must be within the last 7 days.

Fixed!

File: src/content/en/updates/2019/10/chromium-chronicle-7.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+compiler bug.
+
+A few tricks will let a Chromium developer find and execute the command that
+compiles a particular source file, with modifications as needed.
+
+Start by going to your output directory and using autoninja (or ninja) to
+**compile the file of interest** (and any dependencies) **using the `^` suffix**.
+This suffix tells ninja to build the output of the specified `file—version.o`
+in this case. Then, touch the file, and **compile it (and only it) again with
+the -v (verbose) flag** to ninja:
```
Comment (kyarik) [inline src/content/en/updates/2019/10/chromium-chronicle-7.md]: Shouldn't there be backticks around `-v`?

File: src/content/en/updates/2019/10/chromium-chronicle-7.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+<pre class="prettyprint cc-good lang-js">
+..\..\third_party\llvm-build\Release+Asserts\bin\clang-cl.exe /nologo /showIncludes -imsvc ...
+</pre>
+
+This command allows you to compile the file of interest. To get the preprocessed
+output, use the following steps:
+
+**On Linux or OSX, remove the `-o obj/base/base/version.o` block from the end,
+and add -E**. This tells the compiler to print the preprocessed file to stdout.
```
Comment (kyarik) [inline src/content/en/updates/2019/10/chromium-chronicle-7.md]: Shouldn't there be backticks around `-E`?

File: src/content/en/updates/2019/10/chromium-chronicle-7.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+and add -E**. This tells the compiler to print the preprocessed file to stdout. 
+
+Redirect the output to a file, like this:
+{: .compare-better }
+
+<pre class="prettyprint cc-good lang-cpp">
+../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD ... -E >version.i
+</pre>
+
+**On Windows, remove the /showIncludes option** from the beginning (it prints a
```
Comment (kyarik) [inline src/content/en/updates/2019/10/chromium-chronicle-7.md]: Shouldn't there be backticks around `/showIncludes`?

File: src/content/en/updates/2019/10/chromium-chronicle-7.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+line of output for each #include) **and then add /P** in order to preprocess the
```
Comment (kyarik) [inline src/content/en/updates/2019/10/chromium-chronicle-7.md]: Shouldn't there be backticks around `#include` and `/P`?

File: src/content/en/updates/2019/10/chromium-chronicle-7.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+file instead of compiling it. The results will be saved in the current directory
+in version.i:
```
Comment (kyarik) [inline src/content/en/updates/2019/10/chromium-chronicle-7.md]: Shouldn't there be backticks around `version.i`?

**Pull request merged**